File: jscs_array/taskB.js
let arr = [ 1, 2, -5, 8, -3, 9 , -7 ];
let sum1 = 0;
let sum2 = 0;
for(let i in arr) {
    if(arr[i] > 0) {
        sum1 += arr[i];
    }else{
        sum2 += arr[i];
    }
}
console.log(sum1, sum2);

File: jscs_array/taskB_cs/taskB.cs
int[] arr = { 1, 2, -5, 8, -3, 9 , -7 };
int sum1 = 0;
int sum2 = 0;
foreach(int i in arr) {
    if(i > 0) {
        sum1 += i;
    }else{
        sum2 += i;
    }
}
Console.WriteLine(sum1);
Console.WriteLine(sum2);